docker-compose.yml

```diff
@@ -11,8 +11,10 @@ services:
             - python
             - python-sync
             - redis
+            - cherami
         environment:
-            - WAIT_FOR=go,node,python,python-sync
+            - WAIT_FOR=go,node,python,python-sync,cherami
```

Comment:
So the way to deal w readiness of the Cherami server is like so:

1. Add depends_on: cherami from the go layer in docker-compose.yml
2. Define a HEALTH_CHECK in Cherami's Dockerfile which delegates to a script that returns exit code 0 when the Cherami server is ready for business

If needed, you can make the Cherami layer depend_on:cassandra and define a HEALTH_CHECK in the Cassandra Dockerfile as well.

The WAIT_ON env var exists in Crossdock as a way to accomplish this as well, but is deprecated in favor of Docker's now native health checking, which works with Docker Compose's depend_on directive.